client/src/components/CreateForm/CreateForm.js

```diff
@@ -347,23 +341,23 @@ const CreateForm = ({ formType, onSuccess, onError, onClose, tripId }) => {
             ) : null}
 
             <FormControl fullWidth className={classes.formControl}>
-              <Box display='flex' justifyContent='space-between'>
-                <Box display='flex'>
+              <Box display="flex" justifyContent="space-between">
+                <Box display="flex">
                   <input
-                    accept='image/*'
-                    id='icon-button-file'
-                    type='file'
+                    accept="image/*"
```

Comment:
Single quotes are better than double quotes. Change my mind

Reply:
LMAO I agree. In this case tho, the single quotes are for JS code, double quotes for html tags. I used to apply single quotes to everything but then I guess my linter updated and this is the new convention